pallets/oracle/src/lib.rs

```diff
@@ -159,25 +147,25 @@ pub mod pallet {
 	pub struct GenesisConfig<T: Config> {
 		pub max_delay: u32,
 		pub authorized_oracles: Vec<(T::AccountId, Vec<u8>)>,
```

Comment:
We can remove the authorized oracles from the genesis config since we don't need them anymore (and they are not used).

Reply:
Done